src/main/java/net/minecraftforge/common/ForgeConfigSpec.java

```diff
@@ -527,11 +528,20 @@ public LongValue defineInRange(List<String> path, Supplier<Long> defaultSupplier
 
         public Builder comment(String comment)
         {
+            if(comment == null || comment.length() < 1)
+            {
+                comment = "No comment";
+            }
             context.setComment(comment);
             return this;
         }
         public Builder comment(String... comment)
         {
+            if(comment == null || comment.length < 1 || comment[comment.length - 1].length() < 1)
```

Comment:
I don't like the last condition here. You are checking if the last element in the array is an empty string. That would match a comment such as `comment("Line 1", "", "Line 2", "")` that has extra forced line breaks. I think that will cause many false positives.

A better check would be if the length is equal to 1 and the only comment in the array is empty.

Reply:
Good point. I did briefly debate myself over this, but i didn't reach a conclusion. One moment.